rtgui/perspective.cc

```diff
@@ -247,6 +251,13 @@ PerspCorrection::PerspCorrection () : FoldableToolPanel(this, "perspective", M("
     projection_rotate = Gtk::manage (new Adjuster (M("TP_PERSPECTIVE_PROJECTION_ROTATE"), -45, 45, 0.01, 0, ipers_rotate_left, ipers_rotate_right));
     projection_rotate->setAdjusterListener (this);
 
+    camera_scale= Gtk::manage (new Adjuster (M("TP_PERSPECTIVE_CAMERA_SCALE"), .1, 10, 0.01, 0, ipers_rotate_left, ipers_rotate_right));
```

Comment:
Why `.1` and `0.01` on the same line?

Reply:
0.1 is the minimum value for the scale and 0.01 is the  step. Does that answer the question?

Reply:
No. :grin: 

If you look at the lines nearby, there are more `0\.[0-9]+` than `\.[0-9]+`. So the `.1` should read `0.1`, that's all.

Reply:
Fixed.